client_configurator.py

```diff
@@ -301,6 +303,7 @@
                         delegated_management_subnet_id=subnet_id,                     
                         gossip_certificates=managed_cluster_certs,
                         external_seed_nodes=[SeedNode(ip_address=seed_node) for seed_node in seed_nodes],
+                        external_data_centers=[dc_name],
```

Comment:
Should this be `dc_name.split(' ')`?

Reply:
argparse will automatically parse the list of space-delimited datacenter names into an array, so no need.